C#/tests/syntax_test_Strings.cs

```diff
@@ -54,3 +54,14 @@
 /// ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ string.quoted.double
 ///  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ constant.other.guid
 
+"""
+    SELECT *
+    FROM some_table
+/// ^^^^ meta.string.cs string.quoted.triple.cs source.sql keyword.other.dml.sql
+""";
+/// <- meta.string.cs string.quoted.triple.cs punctuation.definition.string.end.cs
+
+"""
+    no sql here
+/// ^^^^^^^^^^^^ meta.string.cs string.quoted.triple.cs meta.string.cs string.quoted.triple.cs - source.sql
```

Comment:
Syntax changes, fix duplicate scopes...


```suggestion
/// ^^^^^^^^^^^^ meta.string.cs string.quoted.triple.cs - source.sql
```